docs/source/versions1.json

```diff
@@ -12,4 +16,4 @@
         "version": "1.0",
         "url": "../1.0/"
     }
-]
+]
```

Comment:
I think the script that generated this needs to write a newline after the JSON dump